### PR TITLE
Graphql fetch places query and favorite type

### PR DESCRIPTION
**Pull request by msakr21**
# Description

Added GraphQL favorite type and fields in query.
Created query that connects api services with field in query type utilizing a specific type (hotel for now)

Fixes # (issue)

Renamed a few parameters or objects in various files to be more descriptive of what they do/are
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Graphiql querying. Still have not started working on implementing integration tests for graphiql

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] Coverage is the same or higher than it was before my changes. If it's lower, it's because: 

## To-Dos

List anything that needs to be looked at or changed in future commits. Please include file path and line number if relevant.
- [ ] push to Heroku for FE integration
- [ ] give list of popular place categories to FE to develop filters
- [ ] modify any references to hotels to places instead and string interpolate front end query data to utilize categories sent
- [ ] set up default category string based on agreed upon popular place categories

### CODE REVIEW

File: app/graphql/queries/fetch_hotels.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      argument :city, String, required: true
+      argument :country, String, required: true
+  
+      #Astrid: refactor using reduce
+      def resolve(args)
+        response = []
+        city_info = GeocodingFacade.city_info(args[:city], args[:country])
+        hotels = PlacesFacade.hotels(city_info)
+
+        hotels.each do |hotel|
```
Comment (Astrid-Hecht) [inline app/graphql/queries/fetch_hotels.rb:14]: I'll change to a reduce enumerable

File: app/services/places_service.rb [hunk 2]
```diff
@@ -3,8 +3,10 @@ def self.conn
     Faraday.new(url: 'https://api.geoapify.com', params: { apiKey: ENV['places_api_key']})
   end
 
-  def self.get_hotels(city)
-    response = conn.get("v2/places?categories=accommodation.hotel&filter=circle:#{city.longitude},#{city.latitude},20000")
+  default_categories = []
+
+  def self.get_hotels(city_info, caetgory = default_categories)
```
Comment (Astrid-Hecht) [inline app/services/places_service.rb]: Typo in category, I'll fix in my PR

File: db/seeds.rb [hunk 3]
```diff
@@ -5,5 +5,10 @@
...
-User.create(name: 'Ash Ketchup', email: 'ash_ketchup@pokemon.com', password: 'derpderpderp')
-User.create(name: 'Mega Man', email: 'mega_man@legends.com', password: 'whatever')
\ No newline at end of file
+user1 = User.create(name: 'Ash Ketchup', email: 'ash_ketchup@pokemon.com', password: 'derpderpderp')
+user2 = User.create(name: 'Mega Man', email: 'mega_man@legends.com', password: 'whatever')
+Favorite.create(user_id: user1.id, ninja_id: "1", place_name: "Cookie Center", thumbnail_url: "bkabkabkabka")
+Favorite.create(user_id: user1.id, ninja_id: "2", place_name: "Poke Center", thumbnail_url: "blablablabla")
+Favorite.create(user_id: user2.id, ninja_id: "1", place_name: "Cookie Center", thumbnail_url: "bkabkabkabka")
+Favorite.create(user_id: user2.id, ninja_id: "2", place_name: "Poke Center", thumbnail_url: "blablablabla")
+Favorite.create(user_id: user2.id, ninja_id: "3", place_name: "some mega man reference", thumbnail_url: "whatever")
```
Comment (Astrid-Hecht) [inline db/seeds.rb:14]: Circuit city?

**Review by Astrid-Hecht — APPROVED** [review]
This looks great, thank you sm for going over this with me and getting all this done

File: db/seeds.rb (hunk 3 above)
Comment (msakr21) [reply db/seeds.rb:14]: Yeah. Make the change!

File: app/services/places_service.rb (hunk 2 above)
Comment (msakr21) [reply app/services/places_service.rb]: Already fixed! circleCI caught the unintentional inclusion of this from our conversation.

**Review by lcole37 — APPROVED** [review]

**Pull request merged**